interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/manipulate-report-elements/arrange-report-controls.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Arrange Report Controls
+author: Anna Gubareva
+---
+# Arrange Report Controls
+
+The following tools allow you to control report elements' size, location, alignment and maintain the distance between them:
+
+* **Rulers**
+	
+	The Report Designer provides horizontal and vertical rulers to help you determine report elements' size and location.
+
+	![](../../../../../images/eurd-win-designer-rulers.png)
+	
+* **Layout Toolbar**
+	
+	The **Layout** toolbar tab enables you to align report controls to each other, change the report element's size, change the stacked elements' order, and so on.
+
+	![](../../../../../images/eurd-win-toolbar-layout-tab.png)
+
+* **Snapping**
+	
+	Use the **Snapping** toolbar group or a report's **Snapping Mode** property to enable automatic report control snapping to a grid and/or snap lines.
+	
+	* **Snap Grid**
+		
+		The design surface displays a visual grid that allows you to determine elements' size and location in a report. Use the **Snap Grid Size** and **Snap Grid Step Count** properties to customize the grid's settings.
+
+		![](../../../../../images/eurd-win-snap-grid-size.png)
+		
+		To hide the grid, disable the **Grid Lines** option in the toolbar's **View** tab or set a report's **Draw the Grid** property to **false**.
+
+		![](../../../../../images/eurd-win-toolbar-show-grid-lines.png)
+		
+		> [!NOTE]
+		> Use the **Report Unit** property to select the [measurement units](../../configure-design-settings/change-a-report's-measurement-units.md) applied to the grid size.
+						
+		Relocating a report control with the mouse or ARROW keys automatically aligns the control to the nearest grid cell.
+
+		You can use the **Align to Grid** toolbar button or context menu item to align the selected controls to the grid's cells.
+
+		![](../../../../../images/eurd-win-align-to-grid-context-menu-item.png)
+		
+		You can temporarily ignore snapping to the grid when you move and resize controls. To do this, hold down ALT if you use the mouse and CTRL if you use the keyboard.
```

Comment:
"hold down ALT if you use the mouse and CTRL if you use the keyboard" There are 2 phrases like this in this doc. The meaning isn't clear- ALT is also a keyboard key?